src/Presentation/Nop.Web/Views/Product/ProductReviews.cshtml

```diff
@@ -52,11 +52,11 @@
                                 <div class="rating-wrapper">
                                     <div class="label first">@T("Reviews.Fields.Rating.Bad")</div>
                                     <div class="rating-options">
-                                        <input asp-for="AddProductReview.Rating" value="1" type="radio" id="addproductrating_1" aria-label="product rating 1" />
-                                        <input asp-for="AddProductReview.Rating" value="2" type="radio" id="addproductrating_2" aria-label="product rating 2" />
-                                        <input asp-for="AddProductReview.Rating" value="3" type="radio" id="addproductrating_3" aria-label="product rating 3" />
-                                        <input asp-for="AddProductReview.Rating" value="4" type="radio" id="addproductrating_4" aria-label="product rating 4" />
-                                        <input asp-for="AddProductReview.Rating" value="5" type="radio" id="addproductrating_5" aria-label="product rating 5" />
+                                        <input asp-for="AddProductReview.Rating" value="1" type="radio" id="addproductrating_1" aria-label="@T("Reviews.Fields.Rating.Bad")" />
+                                        <input asp-for="AddProductReview.Rating" value="2" type="radio" id="addproductrating_2" aria-label="@T("Reviews.Fields.Rating.NotGood")" />
```

Comment:
Could you please share the tool you used? I'm interested why we shoud add aria-label here if this link is self0described

Reply:
Yes we use a combination of the Accessibility audit in Google Chrome, a tool called SortSite, and some of our customers also use a tool called AChecker. Without the aria-label attribute added to each of these inputs, a screen reader is unable to determine what the individual choices are.

Reply:
I'm sorry I commented here by mistake. This question was about Facebook etc links. I added the comment where it's nedded